test/test_functional_tensor.py

```diff
@@ -1067,5 +1018,52 @@ def test_perspective_interpolation_warning(tester):
         tester.assertTrue(res1.equal(res2))
 
 
+@pytest.mark.parametrize('device', ["cpu", ])
+@pytest.mark.parametrize('dt', [None, torch.float32, torch.float64, torch.float16])
+@pytest.mark.parametrize('size', [[96, 72], [96, 420], [420, 72]])
+@pytest.mark.parametrize('interpolation', [BILINEAR, ])
```

Comment:
Thanks for the port! Just 2 nits: we probably don't need the `device` and `interpolation` parametrizations since they only have one element, so I'd suggest to remove them. Should we need them in the future, it will be very  easy to add back

Reply:
True, I agree but as mentioned in the description, in the follow-up PRs we aim to add more interpolation modes and CUDA support, that's why I kept them.